superset/cli/importexport.py

```diff
@@ -176,6 +176,8 @@ def import_datasources(path: str) -> None:
     from superset.commands.dataset.importers.dispatcher import ImportDatasetsCommand
     from superset.commands.importers.v1.utils import get_contents_from_bundle
 
+    g.user = security_manager.find_user(username="admin")
```

Comment:
Please use the `override_user` context manager.

Reply:
@john-bodley I changed this line to `with override_user(user=security_manager.find_user(username="admin")):` as also used in `superset/sql_lab.py`